ember-toucan-core/src/components/form/file-input/delete-button.hbs

```diff
@@ -3,7 +3,7 @@
   type="button"
   aria-label={{@deleteLabel}}
   {{! TODO: remove reset-styles https://github.com/CrowdStrike/ember-oss-docs/issues/17 }}
-  class="reset-styles flex-none bg-transparent"
+  class="reset-styles mr-2 flex-none bg-transparent"
```

Comment:
non-blocking question: Should this class be moved onto the consumer at  `ember-toucan-core/src/components/form/file-input/list-item.hbs`:

```hbs
<Form::FileInput::DeleteButton class="mr-2"
```

so that there isn't built-in margin to the button but rather in the "wrapping" component?